src/Rating/Rating.css

```diff
@@ -8,3 +8,7 @@
   display: flex;
   justify-content: center;
 }
+
+:focus {
```

Comment:
Glad we were able to fix this css issue with the box showing up around our stars. The stars are a little tricky. Maybe there is a better option for future implementation.